covid_xprize/validation/cost_generator.py

```diff
@@ -8,18 +8,33 @@
 import argparse
 
 import numpy as np
+import pandas as pd
 
-from covid_xprize.validation.scenario_generator import get_raw_data
 from covid_xprize.validation.scenario_generator import NPI_COLUMNS as IP_COLUMNS
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 FIXTURES_PATH = os.path.join(ROOT_DIR, 'data')
-DATA_FILE = os.path.join(FIXTURES_PATH, "OxCGRT_latest.csv")
+DEFAULT_GEOS = os.path.join(ROOT_DIR, '..', '..', "countries_regions.csv")
```

Comment:
Not using the test Oxford file anymore, but the official list of countries and regions